SONAR-netCDF4/docs/tableAttitude_sub_group.adoc

```diff
@@ -0,0 +1,54 @@
+|==============================================================================================================================================================================================================
+|Description |Obligation |Comment
+|Group attributes | |
+|:string |O |System description
+|Dimensions | |
+|time | |can be fixed or unlimited length, as appropriate
+|Coordinate variables | |
+|uint64 time(time) | |Time from attitude sensor
+|:axis = "T" | |
+|:calendar = "gregorian" | |
+|:long_name = "Timestamps for attitude data" | |
+|:standard_name = "time" | |
+|:units = "nanoseconds since 1601-01-01 00:00:00Z" | |
+|Variables | |
+|float distance(time) |O |Distance travelled by the platform from an arbitrary location.
+|:long_name = "Distance travelled by the platform" | |
+|:units = "m" | |
+|:valid_min = "0.0" | |
+| | |
+|float heading(time) |MA |Platform heading. Measured clockwise from north.
```

Comment:
Does heading belong in the attitude subgroup, or would it be better in the position subgroup?